dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -226,3 +225,28 @@ CWPKI2049E.useraction=Manually replace the account key pair files as directed in
 CWPKI2050E=CWPKI2050E: Failed to back up the existing account key pair file while refreshing the account key pair. The error is ''{0}''.
 CWPKI2050E.explanation=The existing account key pair file was not be backed up. 
 CWPKI2050E.useraction=Ensure that the directory that contains the existing account key pair file is writable. Review the error message for details on the failure.
+
+CWPKI2051W=CWPKI2051W: The renewBeforeExpiration property is set to {0} which is shorter than the minimum renew time. The renewBeforeExpiration property is reset to {1}.
+CWPKI2051W.explanation=The renewBeforeExpiration property is below the minimum duration to request a new certificate. This could have a negative impact on server performance. 
```

Comment:
We reset it to an acceptable value. But if the customer wants to get rid of the message, they would want to change config. I added `To avoid a warning message, ` -- does that help?